lib/feature_flags.js

```diff
@@ -8,6 +8,7 @@
 // unreleased flags gating an active feature
 exports.prerelease = {
   express5: false,
+  openai_instrumentation: false,
```

Comment:
oh yea we have some unit test that is failing because you added a new feature flag